shell/platform/android/io/flutter/embedding/engine/dart/DartExecutor.java

```diff
@@ -196,6 +196,14 @@ public void send(@NonNull String channel, @Nullable ByteBuffer message, @Nullabl
   public void setMessageHandler(@NonNull String channel, @Nullable BinaryMessenger.BinaryMessageHandler handler) {
     messenger.setMessageHandler(channel, handler);
   }
+
+  /**
+   * Returns the number of pending platform channel callback replies.
+   */
+  public int getNumOfPendingChannelCallbacks() {
```

Comment:
How about we call these methods `getPendingChannelResponseCount()`? Generally in Flutter we avoid truncating terms, and also it might be clearer to refer to them as "responses" because they are responding to messages.

Also, in the javadocs can you elaborate on what this means? Currently the javadoc repeats the terms in the name of the method. If a developer doesn't already know what the method represents, then that developer would probably like to know that:
 - every channel message from Android to Flutter will receive a response, even if its empty
 - Flutter tracks the number of responses its waiting for
 - this method returns that number
 - this method is useful for determining whether or not Flutter is idle

Reply:
Done. PTAL.